floodviz/static/js/gage_map.js

```diff
@@ -0,0 +1,86 @@
+document.addEventListener("DOMContentLoaded", function (event) {
+    "use strict";
+
+    // Define helper functions
+    function degreesToRadians(degrees) {
+        return degrees * Math.PI / 180;
+    }
+
+    function radiansToDegrees(radians) {
+        return radians * 180 / Math.PI;
+    }
+
+    // read in projection information
+    var proj = proj4(FV.mapinfo.proj4string);
+
+    var project = function (lambda, phi) {
+        return proj.forward([lambda, phi].map(radiansToDegrees));
+    };
+
+    project.invert = function (x, y) {
+        return proj.inverse([x, y]).map(degreesToRadians);
+    };
+
+    //Define map projection
+    var projection = d3.geo.projection(project);
+
+    // Give projection initial rotation and scale
+    projection
+        .scale(1)
+        .translate([0, 0]);
+
+
+    //Define path generator
+    var path = d3.geo.path().projection(projection);
+
+    // Read in height and width
+    var height = FV.mapinfo.height;
+    var width = FV.mapinfo.width;
+
+    //Create SVG element
+    var svg = d3.select("body")
+        .append("svg")
+        .attr("width", width)
+        .attr("height", height);
+
+    // read in geojson for sites
+    var site_data = FV.mapinfo.site_data;
+
+    // Calculate bounding box transforms for the sites, we don't care about cutting off the background
```

Comment:
Actually the bounding box is also passed in via config.py, so this will need to be changed.